pkg/v1/providers/config_default.yaml

```diff
@@ -429,7 +429,7 @@ TKG_PROXY_CA_CERT: ""
 TKG_IP_FAMILY:
 
 #! Configure cloud provider permissions for TMC enablement. Only affects AWS at present.
-ENABLE_TMC_CLOUD_PROVIDER_PERMISSIONS: false
+DISABLE_TMC_CLOUD_PERMISSIONS: false
```

Comment:
question: By having `DISABLE_TMC_CLOUD_PERMISSIONS: false`, that means we want `enable` by default. Is that expected ?

Reply:
yes. We are enabling it by default.